src/Servers/Kestrel/test/InMemory.FunctionalTests/Http2/Http2ConnectionTests.cs

```diff
@@ -225,7 +225,7 @@ public async Task StreamPool_StreamIsInvalidState_DontReturnedToPool()
             await StartStreamAsync(1, _browserRequestHeaders, endStream: true);
 
             await ExpectAsync(Http2FrameType.HEADERS,
-                withLength: 37,
+                withLength: 33,
```

Comment:
This is strange to have without anything else?

Reply:
That's what https://github.com/dotnet/aspnetcore/pull/19622 was 🤷‍♂

Reply:
This test is also already quarantined (and remaining quarantined I believe).

Reply:
That test was constantly failing in my PRs because I think its supposed to be 37 :)

Reply:
or 33, whichever the updated value is

Reply:
Yeah, it didn't properly account for HPack. Honestly, it wasn't a flaky test, it was just a completely failing test. We can probably unquarantine it right away with this fix, but I was being cautious and recommended @JamesNK leave the quarantine in place.

Reply:
Sounds good.